backend/src/csv_handler/utils/aws.py

```diff
@@ -32,3 +33,135 @@ def get_secret(secret_name: str):
         # else:
         #     binary_secret_data = get_secret_value_response['SecretBinary']
         #     return {"binary_secret_data": binary_secret_data}
+
+
+def upload_file_to_S3(
+        file_path, bucket, bucket_path, local_file_name, file_name_in_s3
+    ) -> None:
+        try:
+            # Connect to aws s3
+            s3 = boto3.resource('s3')
+
+            # Compile bucket path and file name to generate key
+            key = f'{bucket_path}/{file_name_in_s3}'
+
+            # Compile full path of the uploaded file
+            file_full_path = f'{file_path}/{local_file_name}'
+
+            # Upload the file to s3 bucket
+            s3.meta.client.upload_file(file_full_path, bucket, key)
+            
+
+            # Check if the file exists
+            client = boto3.client('s3')
+            client.head_object(Bucket=bucket, Key=key)
+        except Exception as e:
+            log.error(f'Errors: {e}')
+
+
+
+class ClamAVClient:
+    def __init__(self, file_name, data):
+        if not BUCKET_NAME:
+            raise Exception("Bucket name is not set.")
+        self.bucket_name = BUCKET_NAME
+        self.s3_folder = PROJECT_ENV if PROJECT_ENV != 'local' else 'dev'
+        self.file_name = file_name
+        self.data = data
+
+    def scan(self):
+        res = self.scan_file(self.bucket_name, self.s3_folder, self.file_name, self.data)
+        return res
+    
+    def scan_file(self, bucket_name,s3_folder, file_name, data):
+        result = False
+        try:
+            self.upload_bstring_to_S3_as_file(bucket_name,s3_folder, file_name, data)
+            print(f"File {file_name} is uploaded to S3 bucket {bucket_name}.")
+            for i in range(1,11):
+                sleep(10)
+                res = self.read_file_tags(bucket_name,s3_folder, file_name)
+                if res is not None:
+                    log.info(f"Getting tags is done, after {i} attempts.")
+                    self.delete_file_from_s3(bucket_name,s3_folder, file_name)
+                    log.info(f"File {file_name} is deleted from S3 bucket.")
+                    break
+                print(f"Attempt {i} to get tags is not successful.")
+            av_status = [item['Value'] for item in res if item['Key'] == 'av-status']
+            if len(av_status) > 0:
+                if av_status[0] != 'clean':
+                    result = True
+                    
+        except Exception as e:
+            log.error(f'Errors: {e}')
+        
+        finally:
+            return result
+            
+        
+
+
+
+
+    def get_boto_client(self):
+        boto3.setup_default_session(profile_name='AWSPowerUserAccess-992382470170')
```

Comment:
This needs removing as it will assume the role of the executing Lambda, so no need to specify `profile_name`